controllers/drpolicy_controller.go

```diff
@@ -243,6 +250,36 @@ func s3ProfileValidate(ctx context.Context, apiReader client.Reader,
 	return "", nil
 }
 
+func validateCIDRsFormat(drpolicy *ramen.DRPolicy, log logr.Logger) error {
+	// validate the CIDRs format
+	for i := range drpolicy.Spec.DRClusterSet {
+		cluster := &drpolicy.Spec.DRClusterSet[i]
+		if err := CIDRsFomatValidate(cluster.CIDRs, log); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func CIDRsFomatValidate(cidrs []string, log logr.Logger) error {
+	invalidCidrs := []string{}
+
+	for i := range cidrs {
+		if _, _, err := net.ParseCIDR(cidrs[i]); err != nil {
+			invalidCidrs = append(invalidCidrs, cidrs[i])
+
+			log.Error(err, "validation error")
```

Comment:
logging is not required here? the caller can take care of logging?

Reply:
logging is removed